Episode-Proposal-Template

```diff
@@ -0,0 +1,37 @@
+Episode Proposal Template
+
+# [TITLE]
+
+* Proposal: [PR#n]([link to PR])
+* Authors: [Name]([github profile], ...
+* Review Manager: [Name]([github profile])
+* Status: [Proposed, Pre-production, Production, Post-production, Published](link to episode if published)
```

Comment:
status would be updated overtime just like swift evolution. could be tedious but could be fun :)